controllers/workspace/eventhandlers.go

```diff
@@ -47,7 +47,13 @@ func dwRelatedPodsHandler(obj client.Object) []reconcile.Request {
 }
 
 func (r *DevWorkspaceReconciler) dwPVCHandler(obj client.Object) []reconcile.Request {
-	// Check if PVC is owned by a DevWorkspace (per-workspace storage case)
```

Comment:
Rather than removing this comment, I'd move it back on-top of the code it was [originally above](https://github.com/devfile/devworkspace-operator/pull/1233/files#diff-1c6fd18e74606af2c4edc2a26c6df20b2030af8b9f72e3c6bb28796ee11620bdR57)